packages/config/src/options.js

```diff
@@ -336,6 +336,13 @@ export function getNuxtConfig (_options) {
     consola.warn('build.extractCSS.allChunks has no effect from v2.0.0. Please use build.optimization.splitChunks settings instead.')
   }
 
+  // devModules has been renamed to buildModules
+  if (typeof options.devModules !== 'undefined') {
+    consola.warn('`devModules` has been renamed to `buildModules` and will be removed in nuxt 3.')
```

Comment:
```suggestion
    consola.warn('`devModules` has been renamed to `buildModules` and will be removed in Nuxt 3.')
```
If it seems relevant